backend/src/main/java/ai/verta/modeldb/utils/ModelDBUtils.java

```diff
@@ -66,22 +66,14 @@ private ModelDBUtils() {}
 
   public static Map<String, Object> readYamlProperties(String filePath) throws IOException {
     LOGGER.info("Reading File {} as YAML", filePath);
-    filePath = appendOptionalTelepresencePath(filePath);
+    filePath = CommonUtils.appendOptionalTelepresencePath(filePath);
```

Comment:
I _really_ hate the term `utils` as it gives no information about the package. Could we use a more meaningful name for packages?